hooks/pre-exit

```diff
@@ -2,8 +2,19 @@
 
 set -euo pipefail
 
+[[ -z "${IS_VM_HOST:-}" ]] || {
+    echo "No need to uninstall nvm on the host, skipping nvm removal"
+    return
+}
```

Comment:
👍

Reply:
👍 

Looking at this, it made me think the change makes the plugin less friendly for folks that want to use it outside our use case.

That's okay in that we're building this for us first, but I wonder if we could iterate at some point an make it more generic.

Perhaps, instead of backing some of our setup design in the plugin (that Mac hosts have `IS_VM_HOST` in the environment) we could swap to a dedicated env var e.g. `NVM_BUILDKITE_PLUGIN_SKIP`.